### PR TITLE
Update out of date information on the website

### PR DESCRIPTION
**Pull request by afrittoli**
- Update out-of-date data on the mainpage
- Add project logos to the mainpage
- Update the contribution guide to include SDKs
- Update the roadmap for 2023/2024, remove achieved items, add new ones
- Update the community page with new links to the community repo

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -110,21 +110,29 @@
...
+	Represent events associated with Continuous Operations activities, such as incidents and their resolution
+	{{< /card >}}
 	{{< card header="Core Events" >}}
 	Represent shared abstractions such as Task Runs and Pipeline Runs
 	{{< /card >}}
 {{< /cardpane >}}
 
 <p>
-	There is interest in implementing in many of your favorite CI/CD tools, including ArgoCD, Flux, Jenkins, Jenkins-X, Keptn, Knative, Ortelius, Shipwright, Spinnaker, Tekton. Some of them have started proof-of-concept implementations with CDEvents.
+	CDEvents is implemented or being integrated by many of your favorite SDLC tools, including Argo, Flux, Guac, Harbor, Jenkins, Keptn, Spinnaker, Tekton, Testkube, Tracetest and more.
```
Comment (e-backmark-ericsson) [inline content/en/_index.html:135]: SDLC is not a common term in my neighborhood. Do you mind spelling it out, or add a link to the definition you assume?

File: content/en/community/contribution-guidelines.md [hunk 2]
```diff
@@ -13,7 +13,12 @@ description: >
 <!-- cSpell:locale en-US -->
 # Contribution Guidelines
 
-Thank you for contributing your time and expertise to CDEvents. This document describes the contribution guidelines for the project.
+Thank you for contributing your time and expertise to CDEvents. This document describes the contribution guidelines for the project specification and website:
+
+* [sdk-go](https://github.com/cdevents/sdk-go?tab=readme-ov-file#contributing)
+* [sdk-java](https://github.com/cdevents/sdk-java?tab=readme-ov-file#contributing)
+
+The CDEvents SDKs have dedicated contributing guides in the respective repositories.
```
Comment (e-backmark-ericsson) [inline content/en/community/contribution-guidelines.md:21]: This list feels a bit misplaced here. Should there instead be a ## section about SDKs here?

**Review by e-backmark-ericsson — COMMENTED** [review]
This looks almost great :) Just a couple of small things.

File: content/en/_index.html (hunk 1 above)
Comment (afrittoli) [reply content/en/_index.html:135]: Done

File: content/en/community/contribution-guidelines.md (hunk 2 above)
Comment (afrittoli) [reply content/en/community/contribution-guidelines.md:21]: The text below should be before the list, the links point to the contributing guidelines for the SDKs.
Thanks for noticing this.

**Pull request merged**